### ISSUE TITLE
One Security code

### ISSUE THREAD

**Issue by bobynou** [Zyg0te/Locker#14]
A security code that resets the phone if we force demand

**Comment by Zyg0te** [comment]
If I understand you correctly you're saying you would want the ability to punch in a specific lock screen code that would wipe the device?

**Comment by JasperWeiss** [comment]
That would be an awesome feature

**Comment by Zyg0te** [comment]
I would assume that this would be out of scope for Locker as this seems like something only a lock screen app can handle. I will investigate whether it's possible for admin apps to actually to be notified what unlock code the user attempted to use.

**Comment by JasperWeiss** [comment]
I imagine something like this being really useful for journalists, people in oppressed countries etc.
"Give me the password!"
"Ok, sure"
*phone wiped*

**Comment by Zyg0te** [comment]
I agree that such a feature would be awesome though in your example that journalist might be facing some serious consequences. An even better implementation would be a password that boots the device into a completely separate environment (such as Android For Work, Samsung Knox, etc).

**Comment by JasperWeiss** [comment]
Oh, that'd be even better indeed. Can this be done without a Samsung/LG device?
